Repository/VcsPackageFilter.php

```diff
@@ -288,6 +287,23 @@ private function initInstalledPackages()
     }
 
     /**
+     * Get current MultiConstraint class based on the composer version.
+     *
+     * @param array $constraints    Array list of constraints
+     * @param bool  $useConjunctive
+     *
+     * @return \Composer\Semver\Constraint\MultiConstraint|\Composer\Package\LinkConstraint\MultiConstraint
+     */
+    private function getMultiConstraintInstance(array $constraints, $useConjunctive)
+    {
+        if (class_exists('\Composer\Semver\Constraint\MultiConstraint')) {
+            return new \Composer\Semver\Constraint\MultiConstraint($constraints, $useConjunctive);
```

Comment:
`class_exists` needs the full-namespace name (http://php.net/manual/en/function.class-exists.php#113581). But afterwards, yes you're right.